diff --git a/feeds.conf.default b/feeds.conf.default
index 763a26902e..1935eba0a9 100644
--- a/feeds.conf.default
+++ b/feeds.conf.default
@@ -2,3 +2,4 @@ src-git packages https://git.openwrt.org/feed/packages.git;openwrt-18.06
 src-git luci https://git.openwrt.org/project/luci.git;openwrt-18.06
 src-git routing https://git.openwrt.org/feed/routing.git;openwrt-18.06
 src-git telephony https://git.openwrt.org/feed/telephony.git;openwrt-18.06
+src-git xnn https://github.com/xNNism/xnns-openwrt-feed.git;openwrt-18.06
